tasklist/app/views/tasks/index.html.erb

```diff
@@ -1,11 +1,28 @@
 <h1>Task List</h1>
-<div style="text-align:center;">
-	<table>
-		<% @tasks.each do |task| %>
-		  <tr>
-		    <td class="name"><%= link_to "#{task.name}", "/tasks/#{task.id}" %></td>
-		    <td><%= button_to "Delete", "/tasks/#{task.id}", method: "delete", data: { :confirm => "Are you sure you want to delete this task?" } %></td>
-		  </tr>
-	  <% end %>
-	</table>
-</div>
\ No newline at end of file
+<table>
+	<th colspan="2">Completed?</th><th>Task Name</th><th colspan="2">Update!</th>
+	<% @tasks.each do |task| %>
+	  <tr>
+    	<% if task.complete_date.nil? %>
+    		<td class="status" id="checkbox"><%= link_to "&#9744;".html_safe, "/tasks/complete/#{task.id}", method: :patch %></td>
+    	<% else %> 
+    		<td class="status" id="checkbox"><%= link_to "&#9745;".html_safe, "/tasks/incomplete/#{task.id}", method: :patch %></td>
+			<% end %>
+
+			<% if !task.complete_date.nil? %>
```

Comment:
The indentation here should be cleaned up, as it is making it harder to figure out which `if` lines are associated with which `else` and `end` lines.